src/CabanaPD_Solver.hpp

```diff
@@ -510,13 +511,20 @@ class Solver
                 static_cast<double>( particles.numGlobal() ) *
                 output_frequency / step_time;
 
+            double relative_damage = 0.0;
+            if constexpr ( is_fracture<typename force_model_type::
+                                           fracture_type>::value )
+            {
+                relative_damage = force->totalDamage() / particles.numGlobal();
+            }
+
             _step_timer.reset();
             log( out, std::fixed, std::setprecision( 6 ), step, "/", num_steps,
                  " ", std::scientific, std::setprecision( 2 ), step * dt, " ",
-                 W, " ", std::fixed, _total_time, " ", force_time, " ",
-                 neigh_time, " ", comm_time, " ", integrate_time, " ",
-                 energy_time, " ", output_time, " ", std::scientific,
-                 p_steps_per_sec );
+                 force->totalStrainEnergy(), " ", relative_damage, " ",
```

Comment:
I think we should consider whether we output total damage for non-fracture problems. But this can come with the changes in #239